src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -152,6 +152,7 @@ func NewStartCommand(fs afero.Fs, launcher rp.Launcher) *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("unable to load config file: %s", err)
 			}
+			cfg = cfg.FileOrDefaults()
```

Comment:
Why are we not using the env / flag overrides here?

Reply:
Agree, added. :+1: